README.md

```diff
@@ -37,6 +37,24 @@ hugo server # launch a server and serve it at http://localhost:1313/
 
 - Output: `docs/` is the "build" folder, where GitHub look at to show the [Taiwan Gold Card](https://taiwangoldcard.com/) website
 
+### Running automated tests
+
+All tests are ran automatically via [github workflows](./.github/workflows/tests.yml) when you push a commit. However, you can still run the tests on local by following these steps:
+
+1. First, install node / npm. This is required for the test executable. You may use [Node Version Manager](https://github.com/nvm-sh/nvm):
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+2. At this repo's root, perform `npm install`. This will install `cypress`, the automated test runner.
+
+3. `cypress run` to perform all tests in headless mode / CI mode. 
```

Comment:
@geekyme  maybe this one would work better since cypress isn't installed globally: `./node_modules/cypress/bin/cypress run`? 

an idea for an other PR:   `hugo server` needs to be started to run the tests locally. We could perhaps add a `npm test` command that do both command together?

Reply:
weird i was able to do `cypress run`. Does `npx cypress run` work for you?